lib/components/Divider/Divider.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { View } from './Divider.styles';
+import { ThemeNaturaLight } from '../..';
```

Comment:
Circular reference here. This should be
`import { ThemeNaturaLight } from '../../themes';`